src/components/CircleCiBuilds/index.js

```diff
@@ -31,7 +31,7 @@ class CircleCiBuilds extends Component {
           { 
             this.props.latestBuilds.map((build, i) => 
               <Link 
-                key={"commit-" + i} 
+                key={i + '-' + build.id + '-' + build.branchName + '-' + build.projectName + '-'} 
```

Comment:
Isn't it easier to use `${i}-${build.id}-${build.branchName}-${build.projectName}-` instead? (I'm not quite a fan of string concatenation :D)